mne_bids/write.py

```diff
@@ -331,10 +335,14 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     """
     # get measurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (tuple, list, np.ndarray)):
+    if meas_date is None or anonymize is not None:
+        acq_time = 'n/a'
+    elif isinstance(meas_date, (tuple, list, np.ndarray)):  # noqa: E501
+        # for MNE < v0.20
         acq_time = _stamp_to_dt(meas_date).strftime('%Y-%m-%dT%H:%M:%S')
     else:
-        acq_time = 'n/a'
+        # for MNE >= v0.20
+        acq_time = meas_date.strftime('%Y-%m-%dT%H:%M:%S')
```

Comment:
This does not seem robust to me, because we starting an "else" clause expecting a datetime object ... what if we get some other data that has not been caught by the conditionals before?

the final "else" should be along the lines

> "we have checked everything that would require us to do something ... whatever it is now, we have to set it to 'n/a', because there is no other sensible thing to do"

this connects to my criticism of line 339